draft-ietf-tls-tls13.md

```diff
@@ -1860,6 +1874,9 @@ ClientHello (without modification) except:
 - Optionally adding, removing, or changing the length of the "padding"
   extension {{RFC7685}}.
 
+- Other modifications that may be allowed by an extension defined in the
+  future and present in the HelloRetryRequest
```

Comment:
This is a potentially big change, letting future extensions modify ClientHello2 without a formal Update to the core spec.

Reply:
I think that this is reasonable.  We do this for key_share.

Nit: period.

Reply:
ack.